spec/lib/html_cleaner_spec.rb

```diff
@@ -755,42 +755,42 @@
       Stuff."""
 
       doc = Nokogiri::HTML.fragment(add_paragraphs_to_text(html))
-      doc.xpath("./p[1]/em").children.to_s.strip.should == "em tag." 
-      doc.xpath("./p[2]/strong").children.to_s.strip.should == "strong tag."
-      doc.xpath("./p[3]").children.to_s.strip.should == "Stuff."
+      expect(doc.xpath("./p[1]/em").children.to_s.strip).to eq("em tag.") 
+      expect(doc.xpath("./p[2]/strong").children.to_s.strip).to eq("strong tag.")
+      expect(doc.xpath("./p[3]").children.to_s.strip).to eq("Stuff.")
     end
 
     it "should close unclosed tag withing other tag" do
       pending "Opened bug report with Nokogiri"
       html = "<strong><em>unclosed</strong>"
       doc = Nokogiri::HTML.fragment(add_paragraphs_to_text(html))
-      doc.xpath("./p/strong/em").children.to_s.strip.should == "unclosed"
+      expect(doc.xpath("./p/strong/em").children.to_s.strip).to eq("unclosed")
     end
 
     it "should re-nest mis-nested tags" do
       html = "some <em><strong>text</em></strong>"
       doc = Nokogiri::HTML.fragment(add_paragraphs_to_text(html))
-      doc.xpath("./p[1]/em/strong").children.to_s.strip.should == "text" 
+      expect(doc.xpath("./p[1]/em/strong").children.to_s.strip).to eq("text") 
     end
 
     it "should handle mixed uppercase/lowecase html tags" do
       result = add_paragraphs_to_text("<em>mixed</EM> <EM>stuff</em>")
       doc = Nokogiri::HTML.fragment(result)
-      doc.xpath("./p[1]/em[1]").children.to_s.strip.should == "mixed" 
-      doc.xpath("./p[1]/em[2]").children.to_s.strip.should == "stuff" 
+      expect(doc.xpath("./p[1]/em[1]").children.to_s.strip).to eq("mixed") 
+      expect(doc.xpath("./p[1]/em[2]").children.to_s.strip).to eq("stuff") 
     end
 
     %w(b big cite code del dfn em i ins kbd q s samp
      small span strike strong sub sup tt u var).each do |tag|
       it "should wrap consecutive #{tag} inline tags in one paragraph " do
-        if tag == "sup" || tag == "sub"
-          pending "Opened bug report with Nokogiri"
-        end
+        #if tag == "sup" || tag == "sub"
```

Comment:
Incorrect indentation detected (column 8 instead of 10).<br>Missing space after #.